pages/services/dcos-monitoring/1.0.2/release-notes/index.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+title: Release Notes
+menuWeight: 40
+excerpt: New features, updates, and known limitations
+render: mustache
+model: ../data.yml
+---
+
+# Version v1.0.2
+
+## Updates
```

Comment:
I guess we don't call out new dashboards in release notes? Prior release notes don't mention any. I wonder if we'll be forced to break that consistency if we ever cut a release that only adds dashboards.